### PR TITLE
XT-1541 - Update the readme to call attention to the different ways the ixviewer.js can be accessed

### PR DESCRIPTION
**Pull request by derekgengenbacher-wf**
#### Description:
Update documentation to highlight that the ixviewer aasset can be accessed via CDN, github or built locally.

#### Testing:
CI

**review**:
@Workiva/xt

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by sagesmith-wf — DISMISSED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

File: README.md
Comment (derekgengenbacher-wf) [reply README.md:44]: The link dosnt work, because the next release is technically `0.1.58`. So it wont work until I release, which I was planning on doing right after this merges.

File: README.md
Comment (brettkail-wk) [inline README.md:70]: It seems like it should be this?
> of the JavaScript version `MAJOR` and >= `MINOR.PATCH`.

Well, in English:
1. if a major version bump, then the JSON format is broken, and the new js won't necessarily work with older versions of the generated JSON
2. if a minor version bump, then something was added to the JSON format, so if you generate with some minor version, you have to use at least that minor version for the js.

File: README.md
Comment (brettkail-wk) [inline README.md:61]: Are we certain this URL will work when 1.0.0 is released?  Should we release 1.0.0 and then update this README?

**Review by austinmatherne-wk — DISMISSED** [review]

File: README.md
Comment (derekgengenbacher-wf) [reply README.md:61]: Sure we can do that.

**Review by austinmatherne-wk — APPROVED** [review]

File: README.md
Comment (derekgengenbacher-wf) [reply README.md]: 1.0.0 is released. And the link works correctly.

File: README.md
Comment (sagesmith-wf) [inline README.md:42]: Smallest of nits: Generally you would do `<version_tag>` to make it clear that its not actually multiple values. This is completely fine though

**Review by sagesmith-wf — APPROVED** [review]

**Comment by austinmatherne-wk** [comment]
### Verified

* doc change only

QA +10
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**
